Updated the BusyIndicator KB about interacting with the content behin… (#819)
telerik/maui-docs: knowledge-base/allow-interaction-with-busyindicator-content.md

---

> title: Enabling Interaction with the content behind the BusyIndicator

Reviewer: ```suggestion
title: Enabling Interaction with the Content behind the BusyIndicator
```

description: Learn how to display .NET MAUI BusyIndicator above any content to mark long-running operations and allow interacting with the content behind when not busy.
type: how-to

> page_title: Enabling Interaction with the content behind the BusyIndicator

Reviewer: ```suggestion
page_title: Enabling Interaction with the Content behind the BusyIndicator
```

## Solution

> Telerik .NET MAUI BusyIndicator allows you to indicate that a long-running operation is taking place. When you place the BusyIndicator above other content, the control may prevent the user interaction with that content even when `IsBusy` is `False`.

Reviewer: ```suggestion
Telerik .NET MAUI BusyIndicator allows you to indicate that a long-running operation is taking place. When you display the BusyIndicator over other content, the control may prevent the user interaction with that content even when `IsBusy` is `False`.
```

To allow the interaction with the content behind the busy indicator when its `IsBusy` property is `False`, set `InputTransparent` to `True` and `CascadeInputTransparent` to `False` to the `RadBusyIndicator` instance. In case the BusyIndicator has its `Content` defined, set `InputTransparent` to `True` and `CascadeInputTransparent` to `False` to the main layout wrapped in the `Content` as well.

> The example below shows how to utilize the approach with the BusyIndicator - an Entry control is placed below the BusyIndicator and also BusyIndicator's `Content` is defined. The `InputTransparent` and `CascadeInputTransparent` properties are set for both the `RadBusyIndicator` instance and the Grid to allow interaction with the Entry component behind:

Reviewer: ```suggestion
The example below shows how to utilize the approach with the BusyIndicator. An Entry control is placed below the BusyIndicator and the BusyIndicator's `Content` is defined. The `InputTransparent` and `CascadeInputTransparent` properties are set for both the `RadBusyIndicator` instance and the Grid to allow interaction with the Entry component behind:
```
